src/elfpy/markets.py

```diff
@@ -29,19 +29,26 @@ def __init__(
         self,
         share_reserves,
         bond_reserves,
+        liquidity_pool,
         fee_percent,
         token_duration,
         pricing_model,
         time_stretch_constant=1,
         init_share_price=1,
         share_price=1,
+        end_of_time=None,
```

Comment:
yup it was sketchy. it's useful to have the user close out their position at the last moment, so i can value their return more easily.
instead of that, could 1) have the market class tell the user to close up their positions or 2) get the valuation of unclosed positions
we probably need 2) anyway... just feels like actually closing the position is the easiest way to do that. maybe a wrapper like user.closePositionButNotReallyJustGiveMeItsValuationHehe()